classes/patterns/class-sideimagewithtextandcta.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * Side image with text and CTA pattern class.
+ *
+ * @package P4GBKS
+ * @since 0.1
+ */
+
+namespace P4GBKS\Patterns;
+
+/**
+ * Class Side image with text and CTA.
+ *
+ * @package P4GBKS\Patterns
+ */
+class SideImageWithTextAndCta {
+
+	/**
+	 * Pattern name.
+	 *
+	 * @const string PATTERN_NAME.
+	 */
+	const PATTERN_NAME = 'side-image-with-text-and-cta';
+
+	/**
+	 * Side image with text and CTA pattern constructor.
+	 */
+	public function __construct() {
+		if ( ! function_exists( 'register_block_pattern' ) ) {
+			return;
+		}
+
+		register_block_pattern(
+			self::PATTERN_NAME,
+			[
+				'title'      => __( 'Side image with text and CTA', 'planet4-blocks-backend' ),
+				'keywords'   => [ 'cta', 'side', 'image' ],
+				'categories' => [ 'planet4' ],
+				'content'    => '
+					<!-- wp:media-text -->
```

Comment:
Probably best to add the class name property here too. Apparently WP will add it automatically when inserting and there's no console error/warning. But still seems best to add it in a valid state already.

Reply:
![Screenshot from 2022-02-17 10-22-30](https://user-images.githubusercontent.com/7604138/154445499-1534ce7d-3789-41ac-85ce-a28f654d643b.png)

Reply:
Updated!